### DISCUSSION TITLE
Druid Operator is not getting host.

### DISCUSSION THREAD

**Discussion by emincanoguz11** [apache/airflow#27013] [Q&A] [answered]
Hi there,

I have a question for Druid Operator. I give a druid broker connection in connector in this picture. 

<img width="1756" alt="Screen Shot 2022-10-12 at 15 34 40" src="https://user-images.githubusercontent.com/47830986/195344139-dde77d50-b529-4f7d-be8e-0303c3c626b5.png">

I take a dag like this. 
<img width="337" alt="Screen Shot 2022-10-12 at 15 36 53" src="https://user-images.githubusercontent.com/47830986/195344222-0c5e8f59-5347-42d1-a541-76f7207fa3b9.png">
<img width="361" alt="Screen Shot 2022-10-12 at 15 37 01" src="https://user-images.githubusercontent.com/47830986/195344238-a550a749-249d-4b7c-899a-141d4b821f6e.png">
But I take a this error. 
<img width="1685" alt="Screen Shot 2022-10-12 at 15 37 27" src="https://user-images.githubusercontent.com/47830986/195346207-3735f6ef-31ec-4262-a0ea-9fd67b081125.png">

What is the reason of this error? I can't be debug this code because of I use Airflow in Kubernetes. Also I tried request library but this result is same and also I tried DruidHook but ingestion is not start because that DruidHook haven't got any ingestion parameter. I need a help. 

Thanks for answers.

**Comment by emincanoguz11** [comment]
Maybe this problem is for networking issue or something I don't know. So that I can't open issue for this reason.

**Comment by emincanoguz11** [comment]
P.S I updated Airflow v2.4.0 but I didn't work :(

**Reply by emincanoguz11** [reply]
I solve problem.

**Reply by potiuk** [reply]
Maybe you can post a solution /reason for others who might have similar issues @emincanoguz11 ?

**Reply by emincanoguz11** [reply]
I take PR but I forgot this discussion. Okay I will write problem.

**Reply by emincanoguz11** [reply]
@potiuk Can you check PR please if you have free time? https://github.com/apache/airflow/pull/27174

**Comment by emincanoguz11** [comment, marked as answer]
https://github.com/apache/airflow/issues/27029#issuecomment-1286488277 this problem is solved
